edx_adapt/select/skill_separate_random_selector.py

```diff
@@ -108,7 +108,8 @@ def choose_next_problem(self, course_id, user_id):
                         return ret[0]
 
             #if the user has started the post-test, finish it
-            if len( [x for x in self.data_interface.get_all_interactions(course_id, user_id) if x['problem']['posttest']]) > 0:
+            if len([x for x in self.data_interface.get_all_interactions(course_id, user_id)
```

Comment:
The same as above.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html